examples/kdbget.c

```diff
@@ -8,12 +9,52 @@ int main()
 	KDB *handle = kdbOpen(key);
 
 	kdbGet(handle, myConfig, key);
-	// TODO: check for errors and warnings in key
 
-	keyDel(key);
+	//check for error
+	if(keyGetMeta (key,"error"))
+	{
+		printf("Error occurred: %s\n",
+				keyString(keyGetMeta(key,"error/description")));
+		//fields for more information are listed in the value from
+		//the Key returned by keyGetMeta(key,"error")
+	}
+	//check for warnings
+	if(keyGetMeta (key,"warnings"))
+	{
+		const char * warning_count = keyString (keyGetMeta (key,"warnings"));
+		char buffer[] = "warnings/#00/description";
+
+		do {
+			const Key *warnkey = keyGetMeta (key,buffer);
+			printf ("Warning occurred: %s\n",keyString (warnkey));
+			//fields for more information are listed in the value from
+			//the Key returned by keyGetMeta(key,"warnings/#00")
+			buffer[11]++;
```

Comment:
avoid to hardcode 11, prefer to use a variable, e.g. indexOfUnitPosition indexOfTensPosition